Makefile

```diff
@@ -206,6 +211,8 @@ agent-operator-image:
 	$(docker-build)  -t $(IMAGE_PREFIX)/agent-operator:latest -t $(IMAGE_PREFIX)/agent-operator:$(IMAGE_TAG) -f cmd/agent-operator/$(DOCKERFILE) .
 grafana-agent-crow-image:
 	$(docker-build)  -t $(IMAGE_PREFIX)/agent-crow:latest -t $(IMAGE_PREFIX)/agent-crow:$(IMAGE_TAG) -f cmd/grafana-agent-crow/$(DOCKERFILE) .
+agent-smoke-image:
+	$(docker-build)  -t $(IMAGE_PREFIX)/agent-smoke:latest -t $(IMAGE_PREFIX)/agent-smoke:$(IMAGE_TAG) -f cmd/agent-smoke/$(DOCKERFILE) .
```

Comment:
The growing number of agent-related images on dockerhub are giving me some pause 😞 Can we consider pushing both this and crow to our internal `us.gcr.io/kubernetes-dev` registry instead? 

(They also don't need to support seego)